Gemfile.lock

```diff
@@ -172,37 +172,37 @@ GEM
     powerpack (0.1.1)
     public_suffix (3.0.2)
     puma (3.11.4)
-    rack (2.0.5)
+    rack (2.0.6)
     rack-proxy (0.6.4)
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (5.2.0)
-      actioncable (= 5.2.0)
-      actionmailer (= 5.2.0)
-      actionpack (= 5.2.0)
-      actionview (= 5.2.0)
-      activejob (= 5.2.0)
-      activemodel (= 5.2.0)
-      activerecord (= 5.2.0)
-      activestorage (= 5.2.0)
-      activesupport (= 5.2.0)
+    rails (5.2.2)
```

Comment:
Should I expect a change for this in `Gemfile` also? Updating just a lockfile seems unexpected.

Reply:
Nope, the Gemfile is a set of hints and Gemfile.lock is the dependency resolution. `'~> 5.2.0'` equates to >= 5.2.0 *and* < 5.3.0 which is satisified by 5.2.2. I could bump the Gemfile to `~> 5.2.2` but it seems redundant with the Gemfile.lock.

Reply:
Oh, and the command is just `bundle update <optional gem|omit for all gems>`